src/Utility.php

```diff
@@ -56,9 +56,9 @@ protected static function needsAmountConversion(array $parameters)
         $currencies = [
             'BIF', 'DJF', 'JPY', 'KRW', 'PYG',
             'VND', 'XAF', 'XPF', 'CLP', 'GNF',
-            'KMF', 'MGA', 'RWF', 'VUV', 'XOF',
+            'KMF', 'MGA', 'RWF', 'VUV', 'XOF', 'USD',
```

Comment:
Removed from the latest commit. I'll keep these changes in a branch to use in our local projects.